fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt

```diff
@@ -198,23 +237,87 @@ private fun NormalTabsTabIcon(normalTabCount: Int) {
     }
 }
 
+/**
+ * Banner displayed in multi select mode.
+ *
+ * @param bannerItems List of icons to be displayed in the banner.
+ * @param menuItems List of menu items to be displayed in the menu at the end of the banner. Null if
+ * menu should not be displayed.
+ */
 @Composable
-private fun MultiSelectBanner() {
-    Box(
+private fun MultiSelectBanner(
+    selectedTabs: Int,
+    bannerItems: List<BannerItem> = emptyList(),
+    menuItems: List<MenuItem>? = null,
```

Comment:
I'm questioning whether now is the right time to create an API for making `MultiSelectBanner` customizable. I can see the value in doing such an implementation (as mentioned in the ticket, it could allow us to have multiselection for the other Tabs Tray pages), but I'd rather we keep this Composable as simple as possible for now and refactor it later if the use case(s) for this UI changes down the road, if and when UX or product wishes to expand this functionality. 

@gabrielluong @MatthewTighe, do you have any additional thoughts?

Reply:
@MozillaNoah what are you suggesting would be customizable? Either way, I agree that it would be best to wait until we have an established use case rather than pre-optimize

Reply:
@MatthewTighe, it's in reference to this [BZ comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1816522#c2) (essentially making the content of the banner customizable in any shape or form if, hypothetically, we wanted to add selection to the Private and Synced pages).